### PR TITLE
[DOC] Add custom strategy script to quantization with INC example

### PR DESCRIPTION
**Pull request by anko-intel**
## Description ##
Add script with custom strategy for Intel® Neural Compressor.
Update results to measured on latest MxNet build
(2.0.0b20220902 (commit 3a19f0e50d75fedb05eb558a9c835726b57df4cf))
Results:
image
image
image

### Essentials ###
- [x] PR's title starts with a category (e.g. [BUGFIX], [MODEL], [TUTORIAL], [FEATURE], [DOC], etc)
- [x] Changes are complete (i.e. I finished coding on this PR)
- [ ] All changes have test coverage
- [ ] Code is well-documented

### CODE REVIEW

**Comment by anko-intel** [comment]
@mxnet-bot run ci [unix-gpu]

**Comment by anko-intel** [comment]
@mxnet-bot run ci [unix-gpu]

File: docs/python_docs/python/tutorials/performance/backend/dnnl/dnnl_quantization_inc.md [hunk 1]
```diff
@@ -244,34 +244,42 @@ It is done by
...
 loss criterion. Only the `bayesian` strategy didn't find solution within 500 attempts limit. 
 Although these results may suggest that the `mse` strategy is the best compromise between time spent
 to find the optimized model and final model performance efficiency, different strategies may give 
-better results for specific models and tasks. You can notice, that the most important thing done by INC
+better results for specific models and tasks. For example for ALBERT model there is no solution 
+given by build-in INC strategies. For such situation you can create your custom strategy, similar 
+to this one: 
+[custom_strategy.py](https://github.com/apache/incubator-mxnet/blob/master/example/quantization_inc/custom_strategy.py). 
+You can find description of this particular custom strategy in the Medium article
+(add link to article).
```
Comment (anko-intel) [inline docs/python_docs/python/tutorials/performance/backend/dnnl/dnnl_quantization_inc.md:281]:
```suggestion
```

**Comment by anko-intel** [comment]
@mxnet-bot run ci [centos-gpu, unix-gpu]

**Review by bgawrych — APPROVED** [review]

**Pull request merged**